server/api/crud/files.py

```diff
@@ -77,12 +77,10 @@ def _get_filestat(
 
         logger.debug("Got get filestat request", path=path)
 
-        secrets = secrets or {}
-        secrets.update(server.api.api.utils.get_secrets(auth_info))
-
+        enriched_secrets = self._enrich_secrets_with_auth_info(auth_info, secrets)
```

Comment:
@moranbental I believe that enrichment should follow the common logic across the entire feature. If we change it in artifact deletion, we should change it here as well.

Reply:
Just to clarify - generally @moranbental is correct that this changes the behavior and we want to avoid breaking BC. However the reason this is ok is because the current behavior is a bug and not because we want to align with other flows. Of course aligned flows is something that we want but not in the cost of breaking BC. A good reason the change behavior is a bug 🙂